docs/reference/rest_api.md

```diff
@@ -2257,95 +2278,225 @@ Content-Type: application/json
+  "revision_id": 1,
+  "created": "2021-04-29T14:24:02.806782+00:00",
+  "links": {
+    "self": "{scheme+hostname}/api/communities/my_community_id",
+    "self_html": "{scheme+hostname}/communities/my_community_id",
+    "settings_html": "{scheme+hostname}/communities/my_community_id/settings",
+    "logo": "{scheme+hostname}/api/communities/my_community_id/logo",
+    "rename": "{scheme+hostname}/api/communities/my_community_id/rename"
+  },
+  "metadata": {
+    "title": "My Community",
+    "type": "event",
+    "curation_policy": "This is the kind of records we accept.",
+    "description": "This is an example Community.",
+    "page": "Information for my community.",
+    "website": "https://inveniosoftware.org/",
+    "funding": [
+      {
+        "award": {
+          "number": "246686",
+          "title": "OpenAIRE",
+          "identifier": ".../246686",
+          "scheme": "openaire"
+        },
+        "funder": {
+          "identifier": "00k4n6c32",
+          "scheme": "ror",
+          "name": "European Commission"
+        }
+      }
+    ],
+    "organizations": [
+      {
+        "name": "CERN",
+        "identifiers": [
+          {
+            "identifier": "01ggx4157",
+            "scheme": "ror"
+          }
+        ]
+      }
+    ]
+  }
+}
 ```
 
-### Get a Community
+### Rename a Community
 
-`GET /api/communities/<comid>`
+`POST /api/communities/<comid>/rename`
 
- **Parameters**
+**Parameters**
 
 | Name       | Type   | Location | Description                                                  |
 | ---------- | ------ | -------- | ------------------------------------------------------------ |
-| `comid`       | string | path     | Identifier of the community, e.g.  `my_community`                |
 | `accept`   | string | header   | - `application/json` (default)<br />- `application/vnd.inveniordm.v1+json` |
 
 **Request**
 
 ```http
-GET /api/communities/<comid> HTTP/1.1
-Accept: application/json
+POST /api/communities/<comid>/rename HTTP/1.1
+Content-Type: application/json
+
+{
+  "access": {
+    "visibility": "public",
+    "member_policy": "open",
+    "record_policy": "open"
+  },
+  "id": "new_community_id",
+  "metadata": {
+    "title": "My Updated Community",
+    "description": "This is an example Community.",
+    "type": "event",
+    "curation_policy": "This is the kind of records we accept.",
+    "page": "Information for my community.",
+    "funding": [
+      {
+        "funder": {
+          "name": "European Commission",
+          "identifier": "00k4n6c32",
+          "scheme": "ror"
+        },
+        "award": {
+          "title": "OpenAIRE",
+          "number": "246686",
+          "identifier": ".../246686",
+          "scheme": "openaire"
+        }
+      }
+    ],
+    "organizations": [
+      {
+        "name": "CERN",
+        "identifiers": [
+          {
+            "identifier": "01ggx4157", 
+            "scheme": "ror"
+          }
+        ]
+      }
+    ]
+  }
```

Comment:
I think for the rename only the `id` field is needed. The response does have the full payload response though